src/main/java/com/pub/wysio/entity/Customer.java

```diff
@@ -1,56 +1,32 @@
 package com.pub.wysio.entity;
 
 import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
 
 @Entity
 @Table(name="CUSTOMER")
+@Getter
+@Setter
```

Comment:
technically you don't have lombok in your project on this branch, so if I were to checkout this branch I would have compilation problems. You can leave it like this for now, as the other pr will resolve this issue